docs/Customizing Look and Functionality/Customizing the Template/index.md

```diff
@@ -0,0 +1,114 @@
+# Customizing the Template
+There is some ability to customize the look and feel of the template. How much customization is available depends on the version of Core you are connected with and the design of the head unit.
+
+## Customizing Template Colors (Core 5.0+)
```

Comment:
```suggestion
## Customizing Template Colors
```

Reply:
I've been asking Justin to provide these tags. I think they're more helpful than putting it into the text by being able to tell when features are available rather than guessing.

Reply:
Not sure I agree. There is an implicit guarantee that this feature is available in all versions of Core 5.0+ when tagged as such (IMO), which is not true. Furthermore, it is related to the RPC spec, not versions of Core. It's actual expectation of being available is even more closely tied to the HMI than Core.

Reply:
Then how would you suggest that we tell developers how to tell if the feature is available outside of checking the RPC spec (which is what I was getting at here). Especially in the Remote Control features where code paths diverge between < 6.0 and >= 6.0 and developers have to write code for both if they want to support both.

Reply:
I think checking the RPC spec version is fine, the tag `Core 5.0+` does not directly related to the RPC spec available or what developers need to check.

Reply:
Is replacing it with (RPC 5.0+) acceptable? I'll also add an issue to add a section on how to check the connected RPC spec version, which I don't think is currently available.

Reply:
I think it's more inline with what is required for developers. I'm not a huge fan of in the header overall but I don't have a better suggestion.